remotepin.go

```diff
@@ -0,0 +1,18 @@
+package config
+
+const (
+	PinningTag             = "Pinning"
+	RemoteServicesTag      = "RemoteServices"
+	RemoteServicesSelector = PinningTag + "." + RemoteServicesTag
+)
+
+type Pinning struct {
+	RemoteServices map[string]RemotePinningService
+}
+
+type RemotePinningService struct {
+	ApiEndpoint string
+	ApiKey      string
+	// PinMFS enables watching for changes in MFS and re-pinning the MFS root cid whenever a change occurs.
+	PinMFS bool
```

Comment:
I think this should be:

```
Policies: {
  PinMFS Flag
}
```

Where flag is the config type defined https://github.com/ipfs/go-ipfs/blob/master/docs/config.md#flag and used in this package.

These two changes give us:
1) Cleaner look for multiple policies
2) Can easily tell if the user has set a value